ckanext/canada/helpers.py

```diff
@@ -58,6 +59,12 @@ def user_organizations(user):
     u = User.get(user['name'])
     return u.get_groups(group_type = "organization")
 
+def catalogue_last_update_date():
+    q = model.Session.query(Activity.timestamp).filter(
+        Activity.activity_type.endswith('package')).order_by(
+        Activity.timestamp.desc()).limit(1).one()
```

Comment:
instead of `.limit(1).one()`, which can raise an exception, how about `.first()` and return an empty string when the response `is None`. Helpers shouldn't fail because Jinja has no exception handling so the common pattern is to return an empty string instead.